nservicebus/pipeline/customizing-v6.md

```diff
@@ -50,6 +50,8 @@ The following lists describe some of the common stages which you can build your
 
 ### Outgoing Pipeline Stages
 * Operation specific processing: There is a dedicated stage for each bus operation (e.g. Send, Publish, Subscribe, ...). Behaviors can use one of the following contexts: `IOutgoingSendContext`, `IOutgoingPublishContext`, `IOutgoingReplyContext`, `ISubscribeContext`, `IUnsubscribeContext`.
+* Audit message processing: In here behaviors have access to audited message and audit address. Behaviors should use `IAuditContext` to enlist in this stage.
```

Comment:
I was just missing a `the` in the sentence :)

I guess to be correct its "access to the message to be audited/sent to the audit queue"